app/core/organisation.py

```diff
@@ -60,3 +67,37 @@ def get_organisation_taxonomy_by_name(db: Session, org_name: str) -> TaxonomyDat
         **taxonomy[0],
         "event_types": asdict(entry),
     }
+
+
+def get_organisation_config(db: Session, org: Organisation) -> OrganisationConfig:
+    total = (
+        db.query(FamilyOrganisation)
+        .filter(FamilyOrganisation.organisation_id == org.id)
+        .count()
+    )
+
+    counts = (
+        db.query(Family.family_category, func.count())
+        .join(
+            FamilyOrganisation,
+            Family.import_id == FamilyOrganisation.family_import_id,
+        )
+        .filter(FamilyOrganisation.organisation_id == org.id)
+        .group_by(Family.family_category)
+        .all()
+    )
+    found_categories = {c[0].value: c[1] for c in counts}
+    count_by_category = {}
+
+    # Supply zeros when there aren't any
+    for category in [e.value for e in FamilyCategory]:
+        if category in found_categories.keys():
+            count_by_category[category] = found_categories[category]
+        else:
+            count_by_category[category] = 0
```

Comment:
Potential simplification here, what do you think?

```suggestion
    categories =  [e.value for e in FamilyCategory]
    count_by_category = {c[0].value: (c[1] if c[0].value in categories else 0) for c in counts}
```

Reply:
ok - yes its simpler ... but IMHO this is more difficult to maintain